src/lara-plugin/plugin-api/popup.ts

```diff
@@ -110,6 +112,7 @@ export const addPopup = (_options: IPopupOptions): IPopupController => {
     }
     $dialog.remove();
```

Comment:
You are missing the code after the remove call to update the list, something like this:

`removeList = removeList.filter(fn => fn !== remove);`